openedx/tests/xblock_integration/xblock_testcase.py

```diff
@@ -333,16 +365,23 @@ def setUpClass(cls):
         # So, skip the test class here if we are not in the LMS.
         if settings.ROOT_URLCONF != 'lms.urls':
             raise unittest.SkipTest('Test only valid in lms')
-
         super(XBlockTestCase, cls).setUpClass()
 
+    def setUp(self):
+        """
+        Call setups of all parents
+        """
+        super(XBlockTestCase, self).setUp()
```

Comment:
I'll kill those in a future PR, if you don't mind. This was approved for merge, and all the test are now passing. Thanks for your help!